### PR TITLE
Revise LICENSE

### PR DESCRIPTION
**Pull request by m1stadev**
[Procursus](https://github.com/ProcursusTeam/Procursus), the commonly used bootstrap on coolstar jailbreaks, does offer a cryptocurrency miner. While it is neither maliciously used, nor provides any means of monetary gain for any of the jailbreaks' developers, it technically breaks the 7th clause of the license. I've updated the license to fix this.

### CODE REVIEW

**Comment by coolstar** [comment]
Procursus does not install the cryptocurrency miner by default, so it does not break the license.

**Review by elihwyma — APPROVED** [review]

**Comment by demhademha** [comment]
You may also want to change clause **5**

**Comment by elihwyma** [comment]
> 
> 
> Procursus does not install the cryptocurrency miner by default, so it does not break the license.

> This software (including all forks) may not be hosted on any repo that contains advertisements, donation links, paid/sponsored software, cryptocurrency miners, or other means of monetary gain for the repo owner

Read the license again

**Comment by coolstar** [comment]
@elihwyma right so that's clause 5 not clause 7. wrong clause was edited

File: LICENSE [hunk 1]
```diff
@@ -8,8 +8,8 @@ Redistribution and use in source and binary forms, with or without modification,
...
-5. This software (including all forks) may not be hosted on any repo that contains advertisements, donation links, paid/sponsored software, cryptocurrency miners, or other means of monetary gain for the repo owner
+5. This software (including all forks) may not be hosted on any repo that contains advertisements, donation links, paid/sponsored software, malicious cryptocurrency miners, or other means of monetary gain for the repo owner
 6. This software may not depend on any core packages that are hosted on a repo that violates section #5
-7. This software may not be used on any paid/sponsored jailbreaks, or any jailbreak that contains advertisements, donation links, cryptocurrency miners, or any other means of monetary gain for the jailbreak developer
+7. This software may not be used on any paid/sponsored jailbreaks, or any jailbreak that contains advertisements, donation links, malicious cryptocurrency miners, or any other means of monetary gain for the jailbreak developer
```
Comment (coolstar) [inline LICENSE:13]: This clause does not need editing

**Review by coolstar — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (elihwyma) [reply LICENSE:13]: Seems like you're just changing the license to what suits you best, which is not in the interest of everyone else

**Comment by Diatrus** [comment]
I concur with coolstar's comments.

**Review by elihwyma — CHANGES_REQUESTED** [review]
After further review it appears this clause is also problematic
> or any other means of monetary gain for the jailbreak developer

New and upcoming builds of Taurine are locked behind [Patreon](https://www.patreon.com/coolstar) which is a means of monetary gain for the jailbreak developer ( @coolstar )

**Comment by Diatrus** [comment]
> After further review it appears this clause is also problematic
> 
> > or any other means of monetary gain for the jailbreak developer
> 
> New and upcoming builds of Taurine are locked behind [Patreon](https://www.patreon.com/coolstar) which is a means of monetary gain for the jailbreak developer ( @coolstar )

Incorrect. Taurine the jailbreak app does not contain `advertisements, donation links, cryptocurrency miners, or any other means of monetary gain for the jailbreak developer`, regardless of paid beta status. No essential features of the jailbreak are locked behind a paywall and therefor cannot be considered a “paid/sponsored jailbreak”. The app itself doesn’t even advertise paid builds.

**Comment by Diatrus** [comment]
Closing due to continued irrelevance.

**Comment by nyuszika7h** [comment]
Even if the other clauses do not need to be changed, I'm pretty sure Procursus is violating clause 5, so at least that should be changed.

**Comment by Diatrus** [comment]
@nyuszika7h https://github.com/Sileo/Sileo/commit/3687a9c9c719df8ef2d021f6ba911e4cd8a348c0

**Pull request closed without merging**